reference_conf/README.md

```diff
@@ -0,0 +1,12 @@
+Reference configuration
+-----------------------
+
+The following is a copy of the ``reference.conf`` file matching the version of this documentation.
+It is packaged in the ``java-driver-core`` JAR artifact, and used at runtime to provide the default
+values for all configuration options (in the sources, it can be found under
+``core/src/main/resources``).
+
+See the `configuration page <../manual/core/configuration/>`_ for more explanations.
+
+.. include:: core/src/main/resources/reference.conf
+   :code: properties
```

Comment:
`properties` seems to be the closest syntax for HOCON, as a bare minimum comments should be highlighted.

The code in the generated HTML is correctly lexed into various `span`s, but there are no visible styles, I think we need to add a CSS file.